pkgs/servers/pingvin-share/backend.nix

```
@@ -0,0 +1,51 @@
+{ version
+, src
+}:
+
+{ lib
+, buildNpmPackage
+, vips
+, pkg-config
+, nodePackages
+}:
+
+let
+  prisma = nodePackages.prisma;
+in
+
+buildNpmPackage {
+  pname = "pingvin-share-backend";
+  inherit version;
+
+  src = "${src}/backend";
+
+  postPatch = ''
+    cp ${./backend-package-lock.json} package-lock.json
+  '';
+
+  npmInstallFlags = [ "--build-from-source" ];
+  installPhase = ''
+    cp -r . $out
+  '';
+
+  preBuild = ''
+    ${prisma}/bin/prisma generate
```

prisma should be in nativeBuildInputs only

```suggestion
    prisma generate
```